frontend/src/app/main/ui/viewer.cljs

```diff
@@ -634,5 +635,5 @@
       [:> viewer-content props])
 
     [:div {:class (stl/css :loader-content)}
```

Comment:
Check if this wrapper is needed at all

Reply:
I need to keep a wrappper on the full page loaders because I can't set width and height without setting those for the element dimensions.
Wrapper needs to be 100vh height and 100vw width but the loader has 64px height and 100 px width.